cli/popper/commands/cmd_stages.py

```diff
@@ -0,0 +1,45 @@
+import click
+import os
+import popper.utils as pu
+
+from popper.cli import pass_context
+from os.path import isfile
+
+@click.command('stages', short_help='Comma-separated list of stages to a pipeline')
+@click.argument('pipeline', required=True)
+@click.option(
+    '--set',
+    help="Redefine the stages of a pipeline",
+    required=False
+)
+@pass_context
+def cli(ctx, pipeline, set):
+    """Manipulates the stages of a pipeline.The stages YAML entry specifies
+    the sequence of stages that are executed by the popper check command.When
+    a new pipeline is created, the popper init generates scaffold scripts for
+    setup.sh,run.sh,post-run.sh,validate.sh,teardown.sh by default.(See popper
+    init --help for more)
+
+    Example:
+
+    popper stages #Shows the stages of a pipeline
+
+    popper stages --set=one,two,three #Redefine the stages of a pipeline
+    """
+    config = pu.read_config()
+
+    if not set:
+        pu.info(str(config['pipelines'][pipeline]['stages']))
```

Comment:
Maybe pretty printing this would be better from the user's point of view

Reply:
Good idea. I will look into it.

Reply:
I have sent in a pull request for printing according to the YAML block format (as asked by @ivotron ). 
Just waiting for him to approve it, after that you can use it as function pu.print_yaml  :)

Reply:
agreed!

Reply:
@AnishHota #226 got merged. You can use the function now.